### PR TITLE
Estimated time

### PR DESCRIPTION
**Pull request by andresfulla**

### CODE REVIEW

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/portlet/action/CalculateSegmentsExperimentEstimatedDurationMVCActionCommand.java [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+				segmentsExperiment,
+				ParamUtil.getDouble(actionRequest, "confidenceLevel"),
+				segmentsExperienceKeySplitMap);
+
+		return JSONUtil.put(
+			"segmentsExperimentEstimatedDaysDuration",
+			segmentsExperimentEstimatedDaysDuration);
+	}
+
+	private Long _calculateSegmentsExperimentEstimatedDaysDuration(
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/portlet/action/CalculateSegmentsExperimentEstimatedDurationMVCActionCommand.java:150]: Two methods with the same name but returning different things...For me it's confusing

File: modules/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js [hunk 2]
```diff
@@ -183,6 +250,21 @@ function ReviewExperimentModal({onRun, variants, visible, setVisible}) {
...
+function _variantsToSplitVariantsMap(variants) {
+	return variants.reduce((acc, v) => {
+		return {
+			...acc,
+			[v.segmentsExperimentRelId]: percentageNumberToIndex(v.split)
+		};
+	}, {});
+}
+
+function _getHumanDays(days) {
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js:262]: I'm not sure if it's a good naming...

**Review by dgarciasarai — CHANGES_REQUESTED** [review]

File: modules/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js (hunk 2 above)
Comment (andresfulla) [reply modules/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js:262]: probably not. any suggestions?

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/portlet/action/CalculateSegmentsExperimentEstimatedDurationMVCActionCommand.java (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/portlet/action/CalculateSegmentsExperimentEstimatedDurationMVCActionCommand.java:150]: `_calculateSegmentsExperimentEstimatedDaysDurationJSONObject`

File: modules/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js (hunk 2 above)
Comment (dgarciasarai) [reply modules/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/ReviewExperimentModal.es.js:262]: `getDaysMessage`

**Comment by dgarciasarai** [comment]
Resent here https://github.com/andresfulla/liferay-portal/pull/156

**Comment by dgarciasarai** [comment]
ci:close

**Pull request closed without merging**